README.md

```diff
@@ -10,6 +10,7 @@ The artifacts are published to [bintray](https://bintray.com/linkedin/maven/spar
 - Version 0.1.x targets Spark 2.3 and Scala 2.11
 - Version 0.2.x targets Spark 2.4 and both Scala 2.11 and 2.12
 - Version 0.3.x targets Spark 3.0 and Scala 2.12
+- Version 0.4.x targets Spark 3.2 and Scala 2.12
```

Comment:
Is it possible to allow for scala 2.13 as well?

Reply:
Yeah, I think it actually needs two different builds (they are not compatible in my local mvn builds). However, I think travis isn't being picked up anymore (needs a paid account?). So probably worth to migrate to circleCI or some free alternative.

Reply:
Any chance you can make me an admin so I can set it up?

Reply:
I am not part of the owner group.